### PR TITLE
Make TC js test failures louder

### PR DESCRIPTION
**Pull request by sndrs**
## What does this change?

- adds TC formatter for task runner, making sure errors in asset pipeline/test/lint get loudly reported

## What is the value of this and can you measure success?

- explicit errors on TC

### badly formatted test

local:
screen shot 2016-12-02 at 17 14 45

tc: https://teamcity.gu-web.net/viewLog.html?buildId=25329&buildTypeId=dotcom_pullrequests

### failing JS test

local:
screen shot 2016-12-02 at 17 16 47

tc: https://teamcity.gu-web.net/viewLog.html?buildId=25331&buildTypeId=dotcom_pullrequests

### lint error

local:
screen shot 2016-12-02 at 17 21 11

tc: https://teamcity.gu-web.net/viewLog.html?buildId=25332&buildTypeId=dotcom_pullrequests

## Request for comment

@janua

### CODE REVIEW

File: static/src/javascripts/projects/commercial/modules/dfp/prepare-googletag.js [hunk 1]
```diff
@@ -10,6 +10,7 @@ define([
     'commercial/modules/dfp/dfp-env',
     'commercial/modules/dfp/on-slot-render',
     'commercial/modules/dfp/on-slot-load',
+    'commercial/modules/dfp/ophan-tracking',
```
Comment (sndrs) [inline static/src/javascripts/projects/commercial/modules/dfp/prepare-googletag.js]: designed to fail JS tests

**Comment by janua** [comment]
The error in TC is better, but still a bit unclear!
:+1:

**Comment by sndrs** [comment]
agree, but i think that must be how it's always been...

**Comment by sndrs** [comment]
@janua have updated the description with the final version...

**Review by NataliaLKB — APPROVED** [review]

**Pull request merged**